staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -87,7 +88,11 @@ func newETCD3HealthCheck(c storagebackend.Config) (func() error, error) {
 		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
 		defer cancel()
 		// See https://github.com/etcd-io/etcd/blob/master/etcdctl/ctlv3/command/ep_command.go#L118
-		_, err := client.Get(ctx, path.Join(c.Prefix, "health"))
+		healthPath := path.Join(c.Prefix, "health")
```

Comment:
How about `path.Join("/", c.Prefix, "health")`

Reply:
If c.Prefix already starts with "/", there is no need to prefix one more.

Reply:
The case you mentioned is addressed by the `path.Join()` function. For example, `path.Join("/", "/registry")` outputs `"/registry"` with a single leading "/". See https://github.com/kubernetes/kubernetes/blob/0d9472c2ffccc148cb6caa49b21691c9f98ebf8f/staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go#L101